### DISCUSSION TITLE
Requirements for documentation

### DISCUSSION THREAD

**Discussion by romaintailhurat** [VTL-Community/VTL-Community#42] [General]
Not included: the process for extracting existing documentation

## Source documentation

- Well defined governance
    - Access control, review process
    - Versioning rules
- Use a plain text format in order to enable easy version control
    - Including figures (for example VTL IM)
- Enable multilingual support
- Executable examples
    - Data & metadata & scripts can be used directly

## Process

- Well defined governance
    - Program and scripts maintenance
- Automated process
    - Publication events (commits, tags) start the publication pipeline

## Outputs

- 1 PDF per language
- A multilingual web application
- Style elements according to their usage (table, code)

## Good to have

- For the web version
    - Modifiable and executable code and data

**Comment by amattioc** [comment]
Hi @romaintailhurat ,

thanks for starting this conversation. I think we are done for most of the points in the list. As discussed in our last meeting the missing parts are:

1. **multilingual**: I would move this to the nice to have section for now and tackle it as we have a first official version of the new documentation
2. Some details about governance (e.g. style guidelines or review process for updates).

Last, we still have to cast a decision between the various tools for diagrams (now the finalists are draw.io and PlantUML).
